### PR TITLE
test(e2e): add create content tests

### PR DESCRIPTION
**Pull request by innerdvations**
### What does it do?

- adds test for creating data in the content manager that contains components and dynamic zones
- adds some foundations for testing "advanced options" in CTB (will finish implementing in a follow-up)

### Why is it needed?

Describe the issue you are solving.

### How to test it?

Provide information about the environment and the path to verify the behaviour.

### Related issue(s)/PR(s)

DX-1819
DX-1720 (part 2)

### CODE REVIEW

File: tests/e2e/tests/admin/api-tokens.spec.ts [hunk 1]
```diff
@@ -16,16 +15,18 @@ const createAPIToken = async (page, tokenName, duration, type) => {
...
   await expect(page.getByText('Make sure to copy this token')).toBeVisible();
   await expect(page.getByText('Expiration date:')).toBeVisible();
 };
 
 test.describe('API Tokens', () => {
   test.beforeEach(async ({ page }) => {
-    await resetDatabaseAndImportDataFromPath('with-admin.tar');
-    await page.goto('/admin');
-    await login({ page });
+    await sharedSetup('ctb-edit-st', page, {
```
Comment (innerdvations) [inline tests/e2e/tests/admin/api-tokens.spec.ts:24]: This is just to reduce flakiness since this test kept failing about 20% of the time

**Comment by jhoward1994** [comment]
😢 all the tests look good to me but I'm not sure what's going on with webkit

**Review by butcherZ — APPROVED** [review]

**Comment by 01044206574** [comment]
kmalrhmhfthy@gmail.com

**Comment by 01044206574** [comment]
01044206574

**Comment by 01044206574** [comment]
رحمه فتحي كامل

**Pull request merged**